src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController.cs

```diff
@@ -190,70 +189,47 @@ private void OnSubjectBufferPostChanged(object? sender, EventArgs e)
         {
             AssertIsForeground();
 
-            StartModelUpdateAndSelectedItemUpdateTasks(modelUpdateDelay: TaggerConstants.MediumDelay, selectedItemUpdateDelay: 0, updateUIWhenDone: true);
+            StartModelUpdateAndSelectedItemUpdateTasks(modelUpdateDelay: TaggerConstants.MediumDelay, selectedItemUpdateDelay: 0);
         }
 
         private void OnCaretMoved(object? sender, EventArgs e)
         {
             AssertIsForeground();
-            StartSelectedItemUpdateTask(delay: TaggerConstants.NearImmediateDelay, updateUIWhenDone: true);
+            StartSelectedItemUpdateTask(delay: TaggerConstants.NearImmediateDelay);
         }
 
         private void OnViewFocused(object? sender, EventArgs e)
         {
             AssertIsForeground();
-            StartSelectedItemUpdateTask(delay: TaggerConstants.ShortDelay, updateUIWhenDone: true);
+            StartSelectedItemUpdateTask(delay: TaggerConstants.ShortDelay);
         }
 
         private void OnDropDownFocused(object? sender, EventArgs e)
         {
             AssertIsForeground();
 
-            // Refresh the drop downs to their full information
-            _waitIndicator.Wait(
-                EditorFeaturesResources.Navigation_Bars,
-                EditorFeaturesResources.Refreshing_navigation_bars,
-                allowCancel: true,
-                action: context => UpdateDropDownsSynchronously(context.CancellationToken));
```

Comment:
no more waiting when the drop down is focused (a personal huge pet peeve of mine).  now we simply present what we have.